tests/unit_tests/plugins/modules/output_checkers/test_output_checkers.py

```diff
@@ -0,0 +1,44 @@
+# # # Distribution Statement A. Approved for public release. Distribution unlimited.
+# # #
+# # # Author:
+# # # Naval Research Laboratory, Marine Meteorology Division
+# # #
+# # # This program is free software: you can redistribute it and/or modify it under
+# # # the terms of the NRLMMD License included with this program. This program is
+# # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+# # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+# # # for more details. If you did not receive the license, for more information see:
+# # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+"""Test all Output Checker plugins."""
+import pytest
+
+from geoips.interfaces import output_checkers
+from geoips.commandline import log_setup
+
+log_setup.setup_logging()
+
+
+class TestOutputCheckers:
+    """TestOutputChecker class, defining methods as well."""
+
+    from geoips.interfaces import output_checkers
+
+    available_output_checkers = [
+        str(checker.name) for checker in output_checkers.get_plugins()
+    ]
```

Comment:
That is much more compact!  

Resolves both
https://github.com/NRLMMD-GEOIPS/geoips/pull/379#discussion_r1373555689
and
https://github.com/NRLMMD-GEOIPS/geoips/pull/379#discussion_r1372495781